scripts/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -172,9 +173,12 @@ def create_struct(gene):
     data_summary = pd.DataFrame(
         log_cpm.apply(create_struct, axis=0), columns=['data']
     ).reset_index()
-    data_summary = data_summary.rename({'index': 'gene_symbol'}, axis='columns')
+    data_summary = data_summary.rename({'index': 'gene_name'}, axis='columns')
     # add in cell type info
     data_summary['cell_type_id'] = celltype
+    # add in ENSEMBL IDs
+    gtf = hl.experimental.import_gtf(GENCODE_GTF, force=True, reference_genome='GRCh38', skip_invalid_contigs=True)
```

Comment:
Now that we have `.bgz`, you shouldn't need the `force=True` anymore.

Reply:
Ok, good to know. I've removed it now